src/libs/Navigation/AppNavigator/createCustomBottomTabNavigator/BottomTabBar.tsx

```diff
@@ -90,7 +91,8 @@ function BottomTabBar({selectedTab}: BottomTabBarProps) {
             const currentSearchParams = getCurrentSearchParams();
             if (currentSearchParams) {
                 const {q, ...rest} = currentSearchParams;
-                Navigation.navigate(ROUTES.SEARCH_CENTRAL_PANE.getRoute({query: q, ...rest}));
+                const policy = PolicyUtils.getPolicy(currentSearchParams?.policyIDs);
+                Navigation.navigate(ROUTES.SEARCH_CENTRAL_PANE.getRoute({query: q, ...rest, policyIDs: policy ? currentSearchParams?.policyIDs : undefined}));
```

Comment:
NAB: I think it would have been a bit cleaner/consistent if you had done:

```diff
++                const {q, policyIDs, ...rest} = currentSearchParams;
--                const {q, ...rest} = currentSearchParams;
```